underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -356,23 +468,20 @@ private boolean copy(String src, String dst) {
     LOG.debug("copy from {} to {}", src, dst);
     String strippedSrcPath = stripPrefixIfPresent(src);
     String strippedDstPath = stripPrefixIfPresent(dst);
-    // Retry copy for a few times, in case some Swift internal errors happened during copy.
-    int retries = 3;
-    for (int i = 0; i < retries; i++) {
+    for (int i = 0; i < NUM_RETRIES; i++) {
```

Comment:
could you add back the comment "// Retry copy for a few times, in case some Swift internal errors happened during copy." here? I think it is still valid and helpful